src/status_im/events.cljs

```diff
@@ -547,7 +547,10 @@
 (handlers/register-handler-fx
  :chat.ui/load-more-messages
  (fn [cofx _]
-   (chat.loading/load-more-messages cofx)))
+   (let [chat-id (get-in cofx [:db :current-chat-id])]
```

Comment:
someday we'll have cofx utility ns, with functions like `(get-chat-id cofx)` or `(get-address cofx)`